### ISSUE TITLE
no ckpt

### ISSUE THREAD

**Issue by Psycho-9** [HCIILAB/LAST#4]
FileNotFoundError: [Errno 2] No such file or directory: 'best.pth'
which seems like you didn't provide ckpt of the model?

**Comment by W-Geong** [comment]
Indeed, due to the GitHub file size limit, the ckpt file was not uploaded. I will commit an update with a download link in the near future. You can also try training a ckpt yourself based on the code of BTTR and CoMER.
